### PR TITLE
Add Open Service Mesh to ecosystem

### PR DESCRIPTION
**Pull request by anderseknert**
Also:
* Add some links to Kubernetes authorization item
* Add SPIFFE/SPIRE blog
* Extend Rego tests to verify added/modified YAML files as valid

The last point was intended to be for the integrations.yaml file
only, but thinking more about it made sense not to limit the check
to a single file.

Signed-off-by: Anders Eknert <anders@eknert.com>

### CODE REVIEW

File: .github/workflows/pull-request.yaml [hunk 1]
```diff
@@ -293,5 +293,5 @@ jobs:
         run: |
           curl --silent --fail --header 'Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
           https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/files \
-          | opa eval --data .github/policy/files.rego --format values --stdin-input --fail-defined 'data.files.deny[message]'
+          | opa eval --bundle .github/policy/ --format values --stdin-input --fail-defined 'data.files.deny[message]'
```
Comment (srenatus) [inline .github/workflows/pull-request.yaml:296]: Maybe we should put it somewhere else? It's not really a GitHub thing... 🤔

**Review by srenatus — APPROVED** [review]
LGTM

File: .github/workflows/pull-request.yaml (hunk 1 above)
Comment (anderseknert) [reply .github/workflows/pull-request.yaml:296]: At least currently, the policy is somewhat GitHub specific as it expects the input to be what GH provides at the `/files` endpoint in any given PR. Or is that something you can pull out of plain git JSON-formatted like that when comparing two branches?

Comment (srenatus) [reply .github/workflows/pull-request.yaml:296]: Not exactly what I meant, sorry. The other files in there make sense to GitHub, they're GitHub's API so to speak. The policy bits are not.

Comment (anderseknert) [reply .github/workflows/pull-request.yaml:296]: I don't mind moving the `policy` directory if that's what you mean. Where do you think it should be?

Comment (srenatus) [reply .github/workflows/pull-request.yaml:296]: Hmm .policy? . internal? .ci?

Comment (anderseknert) [reply .github/workflows/pull-request.yaml]: I moved it to the `build` directory as it seemed to be the one to come closest to that.

**Review by srenatus — APPROVED** [review]
Thanks

**Pull request merged**